.gitignore

```diff
@@ -5,6 +5,9 @@
 *.tfstate
 *.tfstate.*
 
+# local lock file
+*.terraform.lock.hcl
```

Comment:
Should this be checked in to pin specific versions of dependencies in CI and surface changes in diffs? Or, would that somehow interfere with users of this module? Maybe dependencies are baked into the CI images already?

Reply:
I was thinking that since this was a module, there was no need to save the lock. We're not actually creating any real resources with our terraform init and tests, so I thought it would be fine to just pull them on demand. I figured that specifying the dependency version was sufficient. Users of this module may very well have varying dependencies.

WDYT?

Reply:
I'll go ahead and add the lock file then. We can always remove it if it turns out it's causing problems.